.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-- 1.5.2
+- 1.6.2
```

Comment:
I'll defer to you all on this decision. I simply rev'd to the version that's common across both our infrastructures.